samples/simd/simd_lib.c

```diff
@@ -0,0 +1,76 @@
+#include "simd_lib.h"
```

Comment:
Better use SIMD types from platform APIs in `uikit` or `objc` samples.

Reply:
> Better use SIMD types from platform APIs in uikit or objc samples.

This would clutter these samples with unrelated SIMD types.

Reply:
I mean use SIMD types from Apple APIs.